spec.bs

```diff
@@ -5763,9 +5776,9 @@ of the following global objects:
     1. Set |finalCompletion| be [=ECMAScript/Completion Record|Completion=]([$Call$](F, `undefined`,
        |arguments|)).
 
-       In |timeout| milliseconds, if the invocation of [$Call$] has not completed,
-       [=immediately=] interrupt the execution, set |finalCompletion| to a new
-       [=ECMAScript/throw completion=] given null, and set |timeoutOccurred| to true.
+       In |timeout| milliseconds less the execution time of [$ScriptEvaluation$], if the invocation

```

Comment:
so https://crsrc.org/c/content/services/auction_worklet/bidder_worklet.cc;drc=835cae18e7fd473637373818af45644c69e364a8;l=1131 is wrong, which needs to substract the time of scriptEvaluation?
```suggestion
       In |timeout| milliseconds minus the execution time of [$ScriptEvaluation$], if the invocation
```

Reply:
No, they share a AuctionV8Helper::TimeLimit which takes care of that

Reply:
oh right. Missed that. Thanks!